pilot/pkg/secrets/kube/multicluster.go

```diff
@@ -36,6 +37,15 @@ type Multicluster struct {
 
 var _ secrets.MulticlusterController = &Multicluster{}
 
+var clustersCount = monitoring.NewGauge(
```

Comment:
Can we combine remote cluster metric to this, and add a cluster_type label, which has remote/local as possible values?

Reply:
yup. i wanted to see if we thought both were needed before getting clever with the aggregation.

Reply:
ok. done. let me know what you think.